lib/ethers/rpc_client.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Ethers.RpcClient do
+  @doc false
+  @spec rpc_client() :: atom()
+  def rpc_client do
+    case Application.get_env(:ethers, :rpc_client, Ethereumex.HttpClient) do
+      Ethereumex.HttpClient -> Ethers.RpcClient.EthereumexHttpClient
+      module when is_atom(module) -> module
+      _ -> raise ArgumentError, "Invalid ethers configuration. :rpc_client must be a module"
+    end
+  end
```

Comment:
Also added this piece of logic here to ensure backward compatibility.
If `Ethereumex.HttpClient` is in the configuration we use its adapter instead.

Reply:
Should we also check that the module is loaded with `Code.loaded?/1`?

Reply:
> Should we also check that the module is loaded with Code.loaded?/1?

No need to do that IMO.